cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2830,6 +2830,103 @@ action.summary_index.category = Authentication
 action.summary_index.alert_name = Authentication - Successful VPN Login Outside Home Country
 
 
+[Authentication - VPN Login Attemp Outside Working Hour]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A VPN Login Attemp Outside Working Hour. \
+\
+Data Collection - VPN data mapped with authentication data-model and has dest_category=vpn_auth.
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | tstats `cs_summariesonly_authentication` count from datamodel=Authentication WHERE Authentication.dest_category="vpn_auth" AND `cs_vpn_indexes` by Authentication.user, Authentication.src, Authentication.action, _time \
+| rename Authentication.* as * \
+| eval time = _time \
+| `cs_generate_time_fields_for_filter(time)` \
+| where time_wday="Saturday" OR time_wday="Sunday" OR time_hour<6 OR time_hour>19 \
```

Comment:
There is a macro called "cs_ad_password_change_outside_working_hour_definition" in macros.conf
Let's maybe combine them together into one macro.

Reply:
The "cs_ad_password_change_outside_working_hour_definition" macro is for splunk default fields. ie. date_*.
I'll create a common macro during the datamodel changes.